winspi/winspimodule.c

```diff
@@ -368,11 +348,9 @@ winspi_spibusinfo_chipselectlinecount(PySpiBusInfoObject *self, PyObject *args)
     PyObject* result = NULL;
     VALIDATE_SPIBUSINFO(self);
 
-    Py_BEGIN_ALLOW_THREADS;
     if (SUCCESS == get_chipselectlinecount_spibusinfo(self->ob_businfo, &val)) {
```

Comment:
Yes. they releases the GIL and thus, you are not allowed to call any python functions in between them.  If you look at get_chipselectlinecout_spibusinfo() (or any of the functions that I took out ALLOW_THREADs), they all call some kind of python function to set the python exception on error, which resulted in a crash. 

(This is one of the times where I wish GitHub would display changes by commits.  I grouped all these ALLOW_THREADs changes in their own commit and had commit comments describing why we don't want them.)